packages/material-ui-styles/src/withStyles/withStyles.d.ts

```diff
@@ -3,15 +3,20 @@ import { PropInjector, CoerceEmptyInterface, IsEmptyInterface } from '@material-
 import * as CSS from 'csstype';
 import * as JSS from 'jss';
 
-export interface CSSProperties extends CSS.Properties<number | string> {
+export interface BaseCSSProperties extends CSS.Properties<number | string> {
+  /**
+   * Used to control if the rule-set should be affected by rtl transformation
+   */
+  flip?: boolean;
```

Comment:
Could drop including this by default, and instead just keep the interface so that users can augment it

Reply:
I would prefer the augmented solution. `flip` is only available if the `jss-rtl` plugin is added, right? This would help in adding types for other plugins.

Reply:
Seems like it. Yeah, it would.